### PR TITLE
[FIX] issue #9 - Add activity indicator while loading motels data from Azure

### PR DESCRIPTION
**Pull request by RicardoBritoBrens**
If anyone want to test i recommend to use Delay method of the Task class like this.

``` c#
Task.Delay(9000); // millisecondsDelay
```

Resources use to solve the issue:

https://github.com/aritchie/userdialogs

It's working!!!!!!!!!!!
Issue #9 done

### CODE REVIEW

File: src/App/CabanasRD/CabanasRD.csproj [hunk 1]
```diff
@@ -41,6 +42,8 @@
     <Folder Include="Framework\APIs\" />
     <Folder Include="Framework\APIs\Models\" />
     <Folder Include="Helpers\" />
+    <Folder Include="Messages\" />
```
Comment (AngelGarcia13) [inline src/App/CabanasRD/CabanasRD.csproj:45]: Este folder debería ir dentro de App/CabanasRD/Resources

File: src/App/CabanasRD/Extensions/TaskExtensions.cs [hunk 2]
```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Threading.Tasks;
+
+namespace CabanasRD.Extensions
+{
+    public static class TaskExtensions
+    {
+        public static async void Await(this Task task, Action completedCallback, Action<Exception> errorCallBack)
```
Comment (AngelGarcia13) [inline src/App/CabanasRD/Extensions/TaskExtensions.cs:8]: Loved it! Veo que sigues a Brian Lagunas jeje

File: src/App/CabanasRD/Messages/DisplayAlertCommonMessages.cs [hunk 3]
```diff
@@ -0,0 +1,14 @@
+﻿using System;
+namespace CabanasRD
+{
+    public static class CommonDisplayAlertMessages
```
Comment (AngelGarcia13) [inline src/App/CabanasRD/Messages/DisplayAlertCommonMessages.cs:4]: Como solución temporal lo veo bien, luego abrimos un issue para mover los strings a un resx file.

File: src/CabanasRD.sln [hunk 4]
```diff
@@ -12,6 +12,9 @@ EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "CabanasRD.XUnitTests", "Tests\CabanasRD.XUnitTests\CabanasRD.XUnitTests.csproj", "{E6285AFF-138B-4513-A5FD-93742F7AED62}"
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "App", "App", "{27826693-7F7E-41EC-8676-42F84C8931D0}"
+	ProjectSection(SolutionItems) = preProject
```
Comment (AngelGarcia13) [inline src/CabanasRD.sln:15]: What is this?? @RicardoBritoBrens creo que se te fue un archivo para el root del shared project, delete this please!

File: src/App/CabanasRD/Messages/DisplayAlertCommonMessages.cs (hunk 3 above)
Comment (RicardoBritoBrens) [reply src/App/CabanasRD/Messages/DisplayAlertCommonMessages.cs:4]: Sounds pretty cool

I didn't know we can add resx file to hold common strings and share between Android and iOS projets.

File: src/App/CabanasRD/Extensions/TaskExtensions.cs (hunk 2 above)
Comment (RicardoBritoBrens) [reply src/App/CabanasRD/Extensions/TaskExtensions.cs:8]: Yes Sir!

File: src/App/CabanasRD/Messages/DisplayAlertCommonMessages.cs [hunk 5]
```diff
@@ -0,0 +1,14 @@
+﻿using System;
+namespace CabanasRD
```
[the remark this answers is not in the document]
Comment (RicardoBritoBrens) [reply src/App/CabanasRD/Messages/DisplayAlertCommonMessages.cs]: was changed

File: src/App/CabanasRD.Android/CabanasRD.Android.csproj [hunk 6]
```diff
@@ -70,14 +70,13 @@
     <PackageReference Include="Xamarin.Forms.Visual.Material">
       <Version>4.7.0.1080</Version>
     </PackageReference>
+      <Version>7.1.0.446</Version>
```
[the remark this answers is not in the document]
Comment (RicardoBritoBrens) [reply src/App/CabanasRD.Android/CabanasRD.Android.csproj]: I think it's already done

**Review by AngelGarcia13 — APPROVED** [review]

**Pull request merged**